ansible_runner/config/_base.py

```diff
@@ -463,6 +471,13 @@ def wrap_args_for_containerization(self, args, execution_mode, cmdline_args):
                 subdir_path = os.path.join(self.private_data_dir, subdir)
                 if not os.path.exists(subdir_path):
                     os.mkdir(subdir_path, 0o700)
+        if self.container_auth_data:
+            # Pull in the necessary registry auth info, if there is a container cred
+            host = self.container_auth_data.get('host')
+            username = self.container_auth_data.get('username')
+            password = self.container_auth_data.get('password')
+            self.registry_auth_path = self._generate_container_auth_file(host, username, password)
+            new_args.extend(["--authfile={}".format(self.registry_auth_path)])
```

Comment:
`--authfile` is not a valid flag for `docker run`. I'm not sure if it is a goal of this project to make it work seamlessly with both `docker` and `podman`. At a minimum, this means we have no way to test this functionality in our current CI environment.

Reply:
This is a good find. If we do choose to merge this, we need to make a note to address this in a follow-up PR ASAP.

Reply:
On second thought, we should just document that this feature only works with Podman.